openassessment/assessment/api/peer.py

```diff
@@ -45,7 +45,7 @@ def submitter_is_finished(submission_uuid, requirements):
         bool
 
     """
-    if requirements is None:
+    if not requirements:
```

Comment:
Is there a reason why we're checking it this way? Generally, if we want to check for `None` we want to be explicit about that because it'll cause weird behavior with `falsy` values.